.github/workflows/dependabot.yml

```
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
```

It's better if it's monthly, to avoid the noise from the PRs
```suggestion
      interval: monthly
```